Python/pywarpx/picmi.py

```diff
@@ -1570,8 +1578,18 @@ def initialize_inputs(self):
         pywarpx.particles.repeated_plasma_lens_period = self.period
         pywarpx.particles.repeated_plasma_lens_starts = self.starts
         pywarpx.particles.repeated_plasma_lens_lengths = self.lengths
-        pywarpx.particles.repeated_plasma_lens_strengths_E = self.strengths_E
-        pywarpx.particles.repeated_plasma_lens_strengths_B = self.strengths_B
+        if self.warpx_strengths_Ex is not None or self.warpx_strengths_Ey is not None:
+            print('Using Ex and Ey, ignoring \'strengths_E\' if provided',flush=True)
```

Comment:
Is this print statement needed, or is this left over from debugging?
BTW, a handy trick in Python is to take advantage of single and double quotes.
```print('Using Ex and Ey, ignoring "strengths_E" if provided',flush=True)```